reactive/kotlinx-coroutines-rx2/src/main/kotlin/kotlinx/coroutines/experimental/rx2/RxChannel.kt

```diff
@@ -36,6 +38,15 @@ public interface SubscriptionReceiveChannel<out T> : ReceiveChannel<T>, Closeabl
 }
 
 /**
+ * Subscribes to this [MaybeSource] and returns a channel to receive elements emitted by it.
+ */
+public fun <T> MaybeSource<T>.open(): SubscriptionReceiveChannel<T> {
```

Comment:
I was thinking about cases where you'd like to do sth with a value emitted from `Maybe` (or do nothing if it is empty). Then you could do sth like

```kotlin
maybe.consumeEach {
  doSth(it)
}
```

But then again you could probably do the same thing with

```kotlin
maybe.await()?.also {
  doSth(it)
}
```
The former code looks nicer and more explicit to me.